src/tsickle.ts

```diff
@@ -2129,3 +2125,7 @@ function combineSourceMaps(
   }
   return tscSourceMapGenerator ? tscSourceMapGenerator.toString() : tscSourceMapText;
 }
+
+function normalizeArray<T>(arr: T[]|undefined): T[] {
```

Comment:
Is this the same as writing
 [
    ...(foo || []),
   ...(bar || []),
]

above?  It might make more sense to inline, I dunno.

Reply:
Yes, good point.